docs/site/tutorials/a_swift_tour.ipynb

```diff
@@ -330,7 +330,7 @@
         "let quotation = \"\"\"\n",
         "    Even though there's whitespace to the left,\n",
         "    the actual lines aren't indented.\n",
-        "    Except for this line.\n",
+        "      Except for this line.\n",
```

Comment:
Based on how this example read, I'm guessing what was intended to show is that this line was indented.